test/neat.js

```diff
@@ -54,7 +54,7 @@ describe('Neat', function () {
     });
     assert.isBelow(results.error, 0.03);
   });
-  it('XOR using efficientMutation', async function () {
+  it.skip('XOR using efficientMutation', async function () {
```

Comment:
Why are we skipping the test?

Reply:
`efficientMutation` currently depends on inconsistent returns from `network.mutate()` and so will always fail. We should disable efficientMutation for now or refactor it immediately.